modules/example/src/smithy4s/example/StreamedObjects.scala

```diff
@@ -88,4 +89,7 @@ object StreamedObjectsGen extends Service.Mixin[StreamedObjectsGen, StreamedObje
   }
 }
 
-sealed trait StreamedObjectsOperation[Input, Err, Output, StreamedInput, StreamedOutput]
+sealed trait StreamedObjectsOperation[Input, Err, Output, StreamedInput, StreamedOutput] {
```

Comment:
The generated methods in this trait (and its implementations) allow to avoid having to generate a huge pattern-match, which seems to help the compiler.